cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -206,7 +206,7 @@ class PredicateFunctionsTest extends DocumentingTest {
     }
     section("single()", "functions-single") {
       p("""The function `single()` returns `true` if the predicate holds for exactly _one_ of the elements in the given list.
-          #`null` is returned if the list is `null` or all of its elements are `null`.""".stripMargin('#'))
+          #`null` is returned if the list is `null` or if one of the comparisons evaluates to `null`.""".stripMargin('#'))
```

Comment:
As expected, this is `null` for all the 6 examples from my experiment, but if you have a `null` and more than one element that fulfills the predicate e.g. `RETURN single(x IN [1, null, 1] WHERE x = 1)` that return `false`.

```suggestion
          #`null` is returned if the list is `null`, or if the predicate evaluates to `null` for at least one element and `true` for max one element.""".stripMargin('#'))
```